client/src/components/FieldAgents.js

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from "react";
+import AddFieldAgent from "./AddFieldAgent";
+import DeleteFieldAgent from "./DeleteFieldAgent";
+import UpdateFieldAgent from "./UpdateFieldAgent";
+
+
+const VIEW = {
```

Comment:
I really like this approach. What inspired it? It'll make our lives a lot easier when we convert to React Router.